PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -365,6 +365,10 @@ def miniAOD_customizeMC(process):
     #also jet flavour must be switched
     process.patJetFlavourAssociation.rParam = 0.4
 
+
+def miniAOD_customizeGEN(process):
+    process.tauGenJets.GenParticles = "prunedGenParticles"
```

Comment:
looking more carefully, why is it even needed for miniGEN?
I do not see any keep statements that would save tauGenJets
and also looking at regular production (reco+miniAOD+validation) config, the tauGenJets are used only in validation and matching to reco taus.
Did I miss something or did it get dragged in as a needed module through some unwanted dependency (recall that the full PAT config is loaded with the current definition of PATGEN)